set-env.js

```diff
@@ -0,0 +1,49 @@
+require("dotenv").config();
+const fs = require("fs");
+const colors = require("colors");
+const targetPath = "./src/environments/environment.ts";
+const targetProdPath = "./src/environments/environment.prod.ts";
+const envConfigFile = `export const environment = {
+  production: false,
+  backendUrl: '${process.env.BACKEND_URL}'
```

Comment:
you don't currently have a backend url so i'd delete lines 8 and 13. I'd also start up the server and make sure that your environment files for production are being generated